### PR TITLE
Replaces all references to "Sephora" with "NSV13"

### PR DESCRIPTION
**Pull request by crossedfall**
This brings us up to speed with our new project name. I have done a test round and didn't experience any issues. 

### Please do not merge any other changes until this is either rejected or merged

### CODE REVIEW

File: sephora/_DEFINES/overmap.dm [hunk 1]
```diff
@@ -1,17 +0,0 @@
-//Weapon modes
```
Comment (Kmc2000) [inline sephora/_DEFINES/overmap.dm]: You deleted this?

**Comment by Kmc2000** [comment]
Beautiful PR, can you compile it?

**Comment by crossedfall** [comment]
@Kmc2000 yup! Did a round where I walked about dicked around with a number of objects. Travis was also able to compile and run the server in its final check.

File: sephora/_DEFINES/overmap.dm (hunk 1 above)
Comment (crossedfall) [reply sephora/_DEFINES/overmap.dm]: Nope. Not sure why it shows that the entire file was remade eve though I didn't touch it. Best guess is that it's a quirk from renaming the directory.

Comment (crossedfall) [reply sephora/_DEFINES/overmap.dm]: For reference, the "new file" is seen a bit further up. 

image

**Pull request merged**